.github/workflows/system-tests-backend.yml

```
@@ -0,0 +1,83 @@
+name: system-tests-backend
+
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  system-test:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    env:
+      SENTRY_URL: http://127.0.0.1:8000
+    strategy:
+      fail-fast: false
+      matrix:
+        sample: [ "sentry-samples-spring-boot-jakarta" ]
+        java: [ "17" ]
+        include:
+          - sample: "sentry-samples-spring-boot"
+            java: "17"
+          - sample: "sentry-samples-spring-boot-webflux-jakarta"
+            java: "17"
+          - sample: "sentry-samples-spring-boot-webflux"
+            java: "17"
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10.5'
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java }}
```

```suggestion
          java-version: '17'
```
could get rid of the java version matrix in theory